app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author_id, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_posts_counter
```

Comment:
**OPTIONAL** -I suggest using the after_save method instead, as it ensures that the update is made only when the data is saved in the database. Please fix this pray
**OPTIONAL** - I suggest making the update_post_counter a private method.